ob-pattern-1/templates/rbac.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@kdanW since this is a single node deployment of each profile, this file is not required.

In this pattern, this Kubernetes Role and RoleBinding are needed to provide permissions for an IS as KM instance to communicate with the Kubernetes API server, to identify its fellow members for clustering via the Kubernetes Service they all belong to.

In other words, this is used by the WSO2 Kubernetes membership scheme [1].

Since, clustering is not required for a single node deployment, let us remove this. This is required when HA comes into play for each profile.

[1]: [WSO2 Kubernetes Membership Scheme](https://github.com/wso2/kubernetes-common/blob/master/kubernetes-membership-scheme/README.md)

Reply:
Understood. I will remove this

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/7d02d48ffa7b4893cf58076f2808e61124c02869